### DISCUSSION TITLE
Strange hole in the model

### DISCUSSION THREAD

**Discussion by Looooopy** [joshreve/dactyl-keyboard#65] [Q&A] [answered]
When generating my version i get a strange hole as below any idea why this happens?

![image](https://user-images.githubusercontent.com/16755871/143501863-413c667f-ed91-4c48-91dc-9ff0782cfead.png)

generate_configuration.py changes I made
```python
...
'ncols':  6, #6,  # key columns
...
'default_1U_cluster': False
...
#################
    ## Switch Hole ##
    #################
'plate_style': 'HS_NUB',

'oled_configurations': {
...
    CLIP': {
            # Common parameters
            'oled_mount_width': 27.0,  # whole OLED width
            ...
            'oled_left_wall_x_offset_override':  38.5,
            ....
            'oled_mount_connector_hole': 5.0,
            'oled_screen_start_from_conn_end': 5.0,
            'oled_screen_length': 15.5,
            'oled_screen_width': 25.0,
            ...
    }
}
...
'controller_mount_type':  'RJ9_USB_WALL',
```

**Comment by Looooopy** [comment, marked as answer]
Look like this solved my problem

https://github.com/joshreve/dactyl-keyboard/blob/master/src/dactyl_manuform.py#L1132
add this on the line below:
`key_place(web_post_bl(), 3, lastrow),`

Result

![image](https://user-images.githubusercontent.com/16755871/143506241-14c199cb-f320-4468-b47a-c92cabc6965c.png)

**Comment by Looooopy** [comment]
@joshreve  is this the correct way of solving it?

**Reply by joshreve** [reply]
Looks spot on.  I'm guessing it was missed because the standard stagger is different.  Good find, I'll check if that change interferes with anything in another config, but that is the right solution for yours.

**Reply by joshreve** [reply]
Found it.  It is corrected in the 1U sized key version, but was missing in the wider upper key default cluster.  Fixed it and your correction does match the 1U version correctly.  Thanks for the find.
